src/components/Header/NetworkSelector.tsx

```diff
@@ -263,6 +269,7 @@ const NETWORK_SELECTOR_CHAINS = [
   SupportedChainId.POLYGON,
   SupportedChainId.OPTIMISM,
   SupportedChainId.ARBITRUM_ONE,
+  SupportedChainId.CELO,
```

Comment:
In v3-info, you placed the L1 chains before the L2 chains. I think this should be done similarly.

Reply:
I hadn't seen @willhennessy's comment above. Instead of changing this, could you resort the v3-info network selector to match this?